.buildkite/scripts/steps/dra-publish.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -uo pipefail
+
+source .buildkite/scripts/bootstrap.sh
+
+# Publish DRA artifacts
+function run_release_manager() {
+    echo "+++ Publishing $BUILDKITE_BRANCH ${WORKFLOW} DRA artifacts..."
+    dry_run=""
+    if [ "$BUILDKITE_PULL_REQUEST" != "false" ]; then
+        dry_run="--dry-run"
+        # force main branch on PR's or it won't execute
+        # because the PR branch does not have a project folder in release-manager
+        BRANCH=main
+    fi
+    docker run --rm \
+        --name release-manager \
+        -e VAULT_ADDR="${VAULT_ADDR_SECRET}" \
+        -e VAULT_ROLE_ID="${VAULT_ROLE_ID_SECRET}" \
+        -e VAULT_SECRET_ID="${VAULT_SECRET}" \
+        --mount type=bind,readonly=false,src="${PWD}",target=/artifacts \
+        docker.elastic.co/infra/release-manager:latest \
+        cli collect \
+        --project $DRA_PROJECT \
+        --branch "${BRANCH}" \
+        --commit "${BUILDKITE_COMMIT}" \
+        --workflow "${WORKFLOW}" \
+        --version "${BEAT_VERSION}" \
+        --artifact-set $DRA_ARTIFACT_SET \
+        $dry_run
+}
+
+chmod -R 777 build/distributions
```

Comment:
Why are we setting `rwx` everywhere for everyone ?

Reply:
The step is not reusing the distribution folder from the previous step. The folder is restored from the BK artifacts, I am not sure if we had to chmod because of that or not. 
We might need to check, given that Marc is on PTO, I will try to test his PR.

Reply:
It turns out we need this in order for the release-manager container to be able to read the build files.